src/modules/auth/webauthn/WebAuthnScreen.js

```diff
@@ -181,6 +197,24 @@ function WebAuthnCard(props) {
 		}
 	}
 
+	// Edit authenticator name
+	const onSubmitKeyName = async (values) => {
+		let response;
+		try {
+			response = await SeaCatAuthAPI.put(`/public/webauthn/${values.id}`, {"name": `${values.name}`});
+			if (response.data.result != 'OK') {
+				throw new Error(t("WebAuthnScreen|Something went wrong, can't update authenticator"));
+			}
+			props.app.addAlert("success", t("WebAuthnScreen|Authenticator successfully updated"));
+		} catch(e) {
+			console.error(e);
```

Comment:
@Aringoaway Is there any particular error message in the error response?

```
e.response.data.message
```
or 
```
e.response.data.result
```
?

If so, we want to display that to the user in alert message.

Reply:
I simulated some errors and add `e` to alert message `props.app.addAlert("danger", e.response.data.message);`

Add an error to the alert message or just display it in the console? If we decide to display it, do we need to add a translation?

![image](https://user-images.githubusercontent.com/65866093/199666291-f82bfdc3-0766-48c2-a809-ea4c72506b2f.png)
![image](https://user-images.githubusercontent.com/65866093/199666532-0abb45bf-c84a-466f-9f4e-fe2ad916eb20.png)

Reply:
@Aringoaway If there is a message for particular error status (4xx) then yes, we should display the error from the service to the user (as your first picture - but please be aware, that we want to display only ONE error message - in this case we dont want to display Something is wrong ...). Please discuss with Robin the error response statuses and messages and implement accordingly. Thank you

Reply:
@byewokko ^^^

Reply:
For this endpoint there are only two specific errors:
- `404 NOT-FOUND` when the webauthn ID doesn't match anything
- `400 VALIDATION-ERROR` when the name doesn't fit the regex pattern